packages/twenty-server/src/engine/core-modules/billing/utils/is-stripe-valid-product-metadata.util.ts

```diff
@@ -41,3 +41,14 @@ const isValidPriceUsageBased = (priceUsageBased?: string) => {
       return false;
   }
 };
+
+const isValidProductKey = (productKey?: string) => {
+  switch (productKey) {
+    case BillingProductKey.BaseProduct:
+      return true;
+    case BillingProductKey.WorkflowNodeExecution:
+      return true;
+    default:
+      return false;
+  }
+};
```

Comment:
nitpick: I see we already wrote similar code above. However, we may need to use a more robust validation mechanism. We could at least do the following:

```ts
return Object.values(BillingProductKey).includes(productKey)
```